nx.json

```diff
@@ -1,108 +1,134 @@
 {
-    "tasksRunnerOptions": {
-        "default": {
-            "runner": "nx/tasks-runners/default",
-            "options": {
-                "cacheableOperations": [
-                    "build",
-                    "test",
-                    "lint",
-                    "check-types",
-                    "package",
-                    "prepare",
-                    "build:json",
-                    "compile-pb-js",
-                    "compile-pb-as"
-                ],
```

Comment:
why did all of these get removed?

Reply:
Not sure - Lerna/NX did something to integrate the two - not sure why or what it did